webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpacePanelHamburgerNavigation.vue

```diff
@@ -127,7 +129,19 @@
               :class="`${applicationIcon(navigation.icon)} icon-default-color icon-default-size`"> </i>
           </v-list-item-icon>
           <v-list-item-content class="mt-n1">
-            {{ navigation.label }}
+            <div class="d-flex align-center justify-space-between">
+              <span>{{ navigation.label }}</span>
+              <v-btn
+                v-if="getUnreadItemsByApplication(navigation.icon) && SpaceWebNotificationsEnabled && displayBadge"
+                class="error-color-background white--text"
+                :class="badgeAnimation && 'zoomOut' || ''"
```

Comment:
We cannot use some vuetify transitions?

Reply:
I didn't find the zoomOut animation in vuetify transition animation list.